### DISCUSSION TITLE
Facing difficulties while integrating search-extensibility-samples with pnp modern search solution

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2539] [Show and tell]
AS per guidelines below, we have deployed the search-extensibility-samples on app catalog and set configuration (added manifest id at each WP component) but still it was working as expected (for out of the box only, custom suggestions etc).

Can any One please help with steps or video how to integrate search-extensibility-samples with main solution i.e. pnp modern search parts will greatly help the community.

**Comment by wobba** [comment]
@FranckyC any tips?

**Reply by FranckyC** [reply]
@GangadharMFord, do you get any error messages? Also make sure you are using the correct version of extensiblity samples and PnP Modern Search core solution. The two solutions must use the same SPFx versions. For instance the latest version of samples must use the 4.8.0+ PnP Modern Search version.

**Comment by GangadharMFord** [comment]
Hi FrankyC, thank you for information and help. I have tested with 4.7.0 version and it's not working. I'm also tested with 4.8.0 version and no issues.
Could you please help us on what version of extensibility samples will be compatible with 4.7.0 version? it will help us.

And kindly also help us on steps with upgrading 4.7.0 version to 4.8.0 ?
